lib/finch.ex

```diff
@@ -80,6 +84,22 @@ defmodule Finch do
   ### Pool Configuration Options
 
   #{NimbleOptions.docs(@pool_config_schema)}
+
+  ## Example
+
+    {
+      Finch,
+      name: MyFinch,
+      pools: %{
+        default: [
+          size: 10,
+          count: 1,
+          protocol: :http1,
+          max_idle_time: :infinity,
+          conn_opts: [transport_opts: [ciphers: :ssl.cipher_suits(:all)]]
```

Comment:
please don’t use :ssl.cipher_suits/1 as it’s going away in OTP 24 :)

Reply:
Updated to used `:ssl.cipuer_suits/2`